swagger/v1/move.yaml

```diff
@@ -44,6 +44,7 @@ Move:
           - prison_recall
           - prison_transfer
           - police_transfer
+          - video_remand_hearing
```

Comment:
Presumably we want this on v2 moves, too?

Reply:
GPWM - I'll update the v2 docs too 👍